scripts.py

```diff
@@ -419,100 +419,45 @@ def process_work(self, work):
         raise NotImplementedError()      
 
 class WorkConsolidationScript(WorkProcessingScript):
+    """Given an Identifier, make sure all the LicensePools for that
+    Identifier are in Works that follow these rules:
 
-    name = "Work consolidation script"
```

Comment:
This script tried to do a lot of the work now done by LicensePool.calculate_work(), so I was able to rip out most of it and just get it to call calculate_work() over and over again.